### DISCUSSION TITLE
Inquiry Regarding Patching (NerdFont) and Maintaining MonoLisa Basic Variant Font

### DISCUSSION THREAD

**Discussion by psadi** [MonoLisaFont/feedback#259] [Q&A] [answered]
I recently acquired the MonoLisa Basic Variant font and have been exploring the Frequently Asked Questions section available at https://www.monolisa.dev/faq

I'm interested in patching the font and keeping it maintained within a private Git repository for personal use. Additionally, I'm considering setting up a workflow to automate the patching process for future releases.

Could you please advise if there are any conflicts or concerns associated with this approach?

Looking forward to your guidance.

**Comment by bebraw** [comment, marked as answer]
Patching and maintaining a private repository for personal use is within the terms of use so feel free to go ahead. I think I'll add a question around this to the FAQ.

**Reply by psadi** [reply]
Thanks for the confirmation!
